app/Http/Resources/TodoResource.php

```diff
@@ -16,22 +16,20 @@ class TodoResource extends JsonResource
     public function toArray($request)
     {
         return [
```

Comment:
Your resource doesnt work with the current data structure in place
Please take a look at the database structure [Click Here](https://docs.google.com/document/d/1ncSNKNsVrd5fLekRpoy2TQYeB-J3g81bFw9poOtO6fU/edit)